shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/entity/Entity.kt

```diff
@@ -35,12 +35,12 @@ data class Quantity(
 
 @Serializable
 data class FirebaseAuthenticationResponse(
-    val kind: String,
-    val idToken: String,
-    val email: String,
-    val refreshToken: String,
-    val expiresIn: String,
-    val localId: String,
+    val kind: String? = null,
```

Comment:
Ah this is my bad, I asked Leke if we could make it non-nullable but didn't realise it needed to be, sorry!

Reply:
no worries